STUDIO/org.openl.rules.webstudio/webapp/pages/modules/administration/settings/index.xhtml

```diff
@@ -541,7 +541,7 @@
                                                                     <h:outputText value="Name:"
                                                                                   rendered="#{not systemSettingsBean.useDesignRepo}"/>
                                                                     <h:inputText value="#{deployConfigRepo.name}"
-                                                                                 disabled="true"
+                                                                                 disabled="true" maxlength="20"

```

Comment:
I guess this validation required for backend as well.

Reply:
We do not want to break the ability to save settings if the repository name was set more than 20 characters earlier. Also, before starting the application in the property file, you can set any repository name, which may also not fit our restrictions. The main problem was that UI looks bad with a long name, but a long repository name itself does not create additional problems.